CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
 ### Changed
 - **cf-core:** [PATCH] Fix OL `padding-left` so numbers aren't cut off in IE
+- **cf-forms:** [PATCH] Updated CF dependencies
+- **cf-pagination:** [PATCH] Updated CF dependencies
```

Comment:
You could say the actual dependencies since it's only one.

Reply:
Oh yeah, now that I'm not preemptively updating cf-icons it is only one. Fixing.